streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -136,20 +136,13 @@ public void flush() {
         checkForException();
```

Comment:
Alternatively I can fix it by not checking for exceptions like that as suggested in the JIRA, but could you double check that's ok? Thanks.